Libraries/LibWeb/HTML/HTMLTitleElement.cpp

```diff
@@ -50,4 +58,28 @@ void HTMLTitleElement::set_text(String const& value)
     string_replace_all(value);
 }
 
+void HTMLTitleElement::consider_propagate_title_update()
+{
+    if (m_first_block_at_ms == 0) {
+        m_first_block_at_ms = MonotonicTime::now_coarse().milliseconds();
+    }
+
+    if (MonotonicTime::now_coarse().milliseconds() - m_first_block_at_ms > timer_unconditional_update_ms) {
```

Comment:
Let's reuse the `MonotonicTime::now_coarse().milliseconds()` part as a separate variable